sdk/openai/Azure.AI.OpenAI/src/Custom/EmbeddingsOptions.Serialization.cs

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable disable
+
+using System.Text.Json;
+using Azure;
+using Azure.Core;
+
+namespace Azure.AI.OpenAI
+{
+    public partial class EmbeddingsOptions : IUtf8JsonSerializable
```

Comment:
Based on my other comment, you can add the same attribute to this class too:
```csharp
[CodeGenSuppress("global::Azure.Core.IUtf8JsonSerializable.Write", typeof(Utf8JsonWriter))]
```

Then, re-generate the code. This attribute will force the generator to skip generating its own `Write` method and should fix the CI pipeline failure.